public/data/tarunmankar.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Tarun Mankar",
+  "type": "personal",
+  "bio": "Full Stack Web Developer (MERN) | Open Source Contributor",
+  "avatar": "https://github.com/tarunmankar.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/tarunmankar",
+      "icon": "github"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://in.linkedin.com/in/tarun-mankar",
```

Comment:
There is a small typo in your profile link! 🛠

```suggestion
      "url": "https://www.linkedin.com/in/tarun-mankar",
```

Reply:
I have resolved all changes please review